actors/user/src/main/java/org/sunbird/user/actors/MigrationActor.java

```diff
@@ -0,0 +1,46 @@
+package org.sunbird.user.actors;
+
+import org.apache.commons.lang3.StringUtils;
+import org.sunbird.actor.core.BaseActor;
+import org.sunbird.actor.router.ActorConfig;
+import org.sunbird.bean.ShadowUser;
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.response.Response;
+import org.sunbird.common.models.util.ActorOperations;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.models.util.LoggerEnum;
+import org.sunbird.common.models.util.ProjectLogger;
+import org.sunbird.common.request.Request;
+import org.sunbird.common.responsecode.ResponseCode;
+import org.sunbird.user.util.MigrationUtils;
+
+
+@ActorConfig(
+        tasks = {"migrateUser","rejectMigration"},
+        asyncTasks = {}
+)
+public class MigrationActor extends BaseActor {
```

Comment:
use TenantMigrationActor - this class definition is saying : it is used for moving custodian user to tenant user.

Reply:
done